src/main/resources/l10n/JabRef_en.properties

```
@@ -503,7 +503,7 @@ Docs\ (This\ Year)=Docs (This Year)
 Incorrect\ ISSN\ format=Incorrect ISSN format
 Error\ accessing\ catalog=Error accessing catalog
 
-Check\ for\ latest\ version\ online=Check for latest version online
+Check\ for\ updates\ automatically\ on\ startup=Check for updates automatically on startup
```

This sounds like a hendiadys to: "Check automatically on startup" - what else? Enable "Check manually on startup?"
```suggestion
Check\ for\ updates\ on\ startup=Check for updates on startup
```